<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailedFight</class>
 <widget class="QDialog" name="DetailedFight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>711</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>49</y>
     <width>20</width>
     <height>291</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>711</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Username">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Enemyname">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enemy Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Type">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <pointsize>37</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Type Of Fight</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_User">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>331</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Level</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>XP</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Money</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Dead</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Talent</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_Enemy">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>331</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Level</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>XP</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Money</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Dead</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Talent</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gains</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gains</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>330</y>
     <width>721</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>540</y>
     <width>721</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_User_Win">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>WIN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Enemy_Win">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>500</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>WIN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>490</y>
     <width>721</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>500</y>
     <width>20</width>
     <height>51</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>681</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Carlito</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Team / Farmer Talent: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_O_T_U">
          <property name="font">
           <font>
            <family>Carlito</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>None</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Carlito</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Team / Farmer XP: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_O_X_U">
          <property name="font">
           <font>
            <family>Carlito</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>None</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Carlito</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Team / Farmer Talent: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_O_T_E">
          <property name="font">
           <font>
            <family>Carlito</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>None</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Carlito</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Team / Farmer XP: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_O_X_E">
          <property name="font">
           <font>
            <family>Carlito</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>None</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_Link">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>550</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
     <underline>true</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>&lt;a href=&quot;Leekwars.com&quot;&gt;Full Repport&lt;/a&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>line_4</zorder>
  <zorder>line_2</zorder>
  <zorder>line</zorder>
  <zorder>line_3</zorder>
  <zorder>label_Username</zorder>
  <zorder>label_Enemyname</zorder>
  <zorder>label_Type</zorder>
  <zorder>tableWidget_User</zorder>
  <zorder>tableWidget_Enemy</zorder>
  <zorder>label_1</zorder>
  <zorder>label_2</zorder>
  <zorder>line_5</zorder>
  <zorder>label_User_Win</zorder>
  <zorder>label_Enemy_Win</zorder>
  <zorder>line_6</zorder>
  <zorder>line_7</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>label_Link</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
